src/electron/views/results/test-view.tsx

```diff
@@ -19,6 +19,7 @@ export type TestViewProps = {
     cardsViewData: CardsViewModel;
     userConfigurationStoreData: UserConfigurationStoreData;
     contentPageInfo: ContentPageInfo;
+    showTabStops: boolean;
```

Comment:
I'm not a fan of this name. "showTabStops" sounds like it controls an action, when it reflects whether or not tab stops are enabled on the device/toggle. Could we call it "tabStopsEnabled" or something like that, to better capture the notion of what it represents?